website/docs/components/icon.mdx

```diff
@@ -33,35 +34,66 @@ import { Icon } from "@chakra-ui/core"
 
 ## Usage
 
-Use an icon by passing the `name` prop. This name must match an icon key in
-`theme.icons`. By default, the icon inherits the `fontSize` and `color` of its
-parent.
+Use an icon by directly importing it with its respective name as mentioned in
+[next section](#all-icons). By default, Icon component inherits `fontSize` and
+`color` of its parent.
 
 ```jsx
-<Stack direction="row">
+<HStack>
   {/* Default size is 1em => 16px */}
   <PhoneIcon />
 
-  {/* Use the `size` prop to change the icon size */}
-  <CheckCircleIcon size="24px" />
+  {/* Use the `boxSize` prop to change the icon size */}
+  <CheckCircleIcon boxSize="24px" />
 
   {/* Use the `color` prop to change the icon color */}
   <WarningIcon boxSize="32px" color="red.500" />
-</Stack>
+</HStack>
 ```
 
 ### All Icons
 
-Here's a list of the default icons Chakra UI comes with and their respective
-`name`. You can add your own icons as well;
-[see the guide](#adding-custom-icons).
+Here's the list of default icons that comes with Chakra UI with their respective
+`name`,
 
 <IconsList />
 
-## Using an icon library
+## Using an Icon library
```

Comment:
I think the below explanation is required for the Icon component to understand the true purpose of using this component.